invenio_app_rdm/users_ui/views/dashboard.py

```diff
@@ -8,7 +8,7 @@
 
 """RDM User dashboard views."""
 
-from flask import render_template
+from flask import g, render_template
```

Comment:
check the two files above this comment (for some reason I couldn't comment there):
I think this needs to be reverted, somehow the file was replaced with a symlink